.github/workflows/test_build.yml

```
@@ -61,6 +62,7 @@ jobs:
 
     - name: 🔨 Build
       run: |
+        export SKIP_THIRD_PARTY_VALIDATION=1
```

```suggestion
        export SKIP_THIRD_PARTY_VALIDATION="1"
```
just to be the same as the line above